src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --color-primary: #00acdc;
+  /* nav__link settings without margin */
+  --nav__link__padding: 22.5px 0;
+  --nav__link__font-family: roboto, sans-serif;
+  --nav__link__font-size: 12px;
+  --nav__link__font-weight: 500;
+  --nav__link__text-decoration: none;
+  --nav__link__color: black;
+  --nav__link__text-transform: uppercase;
+  --nav__link__display: block;
+  /* nav__link settings without margin */
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+  position: absolute;
```

Comment:
```suggestion

```
There is no need to change a document flow

Reply:
what does it mean?

Reply:
document flow